build/creator/VaultTracker.sol

```diff
@@ -246,6 +211,14 @@ contract VaultTracker {
     return true;
   }
 
+  /// @notice Return both the current maturityRate if it's > 0 (or exchangeRate in its place) and the Compounding exchange rate
+  /// @dev While it may seem unnecessarily redundant to return the exchangeRate twice, it prevents many kludges that would otherwise be necessary to guard it
+  /// @return maturityRate, exchangeRate if maturityRate > 0, exchangeRate, exchangeRate if not.
+  function rates() public view returns (uint256, uint256) {
+    uint256 exchangeRate = Compounding.exchangeRate(protocol, cTokenAddr);
```

Comment:
I dont think this is correct, the `exchangeRate` being compared needs to be the vault.exchangeRate, not the current exchangeRate, right?

Im workin my way through this / have a couple spots im confused by though so maybe im overlooking something